shex/questionnaireShape.shex

```diff
@@ -9,36 +9,32 @@ PREFIX prov: <http://www.w3.org/ns/prov#>
 
 :personShape IRI { 
   a [obo:NCBITaxon_9606];
-  obo:RO_0000056 @:qualityQuestionnaireShape;
-  obo:RO_0000056 @:whodasQuestionnaireShape
+  obo:RO_0000056 @:questionnaireShape
 }
 
-:qualityQuestionnaireShape IRI {
-  a [obo:NCIT_C121522]
+:questionnaireShape IRI {
+  a [obo:NCIT_C121522 ncit:C130322]
```

Comment:
I found that `obo:NCIT_C130322` (http://purl.obolibrary.org/obo/NCIT_C130322) is also resolvable.  May I know the decision of these choices? 

I have encountered such a similar dilemma.

Reply:
Oh no I didn't realize that two different prefixes are used for `NCIT`. But it is excellent bug finding by you :+1: . I guess the chances of having this same bug in other `ShEx` files is very likely. Can you create issue for this bug?